selfdrive/monitoring/driver_monitor.py

```diff
@@ -99,17 +102,22 @@ def __init__(self):
     self.cfactor = 1.
 
 class DriverStatus():
-  def __init__(self, rhd=False):
+  # pylint: disable=no-member
+  def __init__(self, rhd=False, **kwargs):
+    # init settings
+    self.__dict__.update(DRIVER_MONITOR_SETTINGS)
+    self.__dict__.update((k, v) for k, v in kwargs.items() if k in DRIVER_MONITOR_SETTINGS.keys())
```

Comment:
seems like this is for tests? leave a comment so it's clear what this is for, because nothing in openpilot seems to use it

Reply:
also, if this is for tests, think it makes more sense to pass in the class of constants as an argument instead of this update thing